package.json

```diff
@@ -88,9 +88,9 @@
     "zulip-markdown-parser": "^1.0.6"
   },
   "devDependencies": {
-    "@babel/core": "^7.11.4",
+    "@babel/core": "^7.12.9",
     "@babel/preset-env": "^7.11.0",
-    "@babel/runtime": "^7.8.4",
+    "@babel/runtime": "^7.12.5",
```

Comment:
deps: Follow RN in updating some dev dependencies.

Worth mentioning somewhere in the commit message that this doesn't change any of the versions we actually end up using -- our `yarn.lock` already contained versions as new as all of these. It just bumps the explicit lower bounds in the `package.json`.